testsuite/integration-tests/pom.xml

```diff
@@ -623,6 +623,50 @@
 
                     </executions>
             </plugin>
+
+            <!-- generate archive needed by project resteasy-quarkus-testsuite -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <descriptors>
+                        <descriptor>assembly/assembly.xml</descriptor>
+                    </descriptors>
+                    <finalName>resteasy-integration-tests-${project.version}-sources</finalName>
+                    <appendAssemblyId>false</appendAssemblyId>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>trigger-assembly</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-install-plugin</artifactId>
+                <version>3.0.0-M1</version>
```

Comment:
Is specifying the version here really needed ? IOW, does the maven-install-plugin version controlled by current jboss-parent work as well? In that case, I think it would be better not to override the version here.